README.md

```diff
@@ -118,6 +118,49 @@ Here is an example of the default file format that go-i18n supports:
 
 To use a different file format, write a parser for the format and add the parsed translations using [AddTranslation](https://godoc.org/github.com/nicksnyder/go-i18n/i18n#AddTranslation).
 
+Flat Format
```

Comment:
How do you offer to implement this? I propose to add a new boolean flag called `-flat` to output data in flat format